src/idl_gen_kotlin.cpp

```diff
@@ -1299,19 +1317,22 @@ class KotlinGenerator : public BaseGenerator {
   }
   static void GenerateGetterOneLine(CodeWriter &writer, const std::string &name,
                                     const std::string &type,
-                                    const std::function<void()> &body) {
+                                    const std::function<void()> &body,
+                                    bool gen_experimental_unsigned_types = false) {
```

Comment:
Why getters and setters need to be aware of the annotation? Isn't the class itself annotated not enough?